src/Workleap.DomainEventPropagation.Subscription.PullDelivery.Tests/EventPullerTests.cs

```diff
@@ -0,0 +1,270 @@
+﻿using AutoBogus;
+using Azure.Messaging;
+using FakeItEasy;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Workleap.DomainEventPropagation.EventGridClientAdapter;
+
+namespace Workleap.DomainEventPropagation.Subscription.PullDelivery.Tests;
+
+// Test class cannot be made static
+#pragma warning disable CA1052
```

Comment:
Pense si tu mets `StartWaitAndStop` pas static t'auras pu le warning

Reply:
The class has been converted to an `abstract class`; this warning was no longer necessary to disable.